src/Mount.cpp

```diff
@@ -2790,6 +2817,16 @@ void Mount::calculateRAandDECSteppers(long& targetRASteps, long& targetDECSteps)
     float const RALimitR = (RA_LIMIT_LEFT * stepsPerSiderealHour);  
   #endif
 
+  if (pSolutions != NULL)
+  {
+    *(pSolutions + 0) = long(-moveRA);
```

Comment:
mmmm don't do pointer arithmatic please. `pSolutions[0]`

Reply:
If you insist. What's bad about pointer arithmetic?

Reply:
It's error prone and can lead to some nasty bugs. 99.9% of the time you can accomplish what you need with code that can be length-checked at compile time

Reply:
Fixed